api4/plugin_test.go

```diff
@@ -663,7 +663,7 @@ func TestGetInstalledMarketplacePlugins(t *testing.T) {
 
 		plugins, resp = th.SystemAdminClient.GetMarketplacePlugins(&model.MarketplacePluginFilter{})
 		CheckNoError(t, resp)
-		newPlugin.InstalledVersion = manifest.Version
+		newPlugin.InstalledVersion = ""
```

Comment:
Previously, `manifest.Version` /was/ the empty string, but I fixed the bundle on disk, and this test started failing. The intention here was to remove the `InstalledVersion`, since it was removed just above.